### DISCUSSION TITLE
远程控制的时候，MAA界面白屏

### DISCUSSION THREAD

**Discussion by TlgYuri** [MaaAssistantArknights/MaaAssistantArknights#5772] [General]
是就这么设计的，还是说是我的问题吗？我用家里电脑wsa挂肉鸽隔三岔五闪退，这样不太方便上班的时候远程摸🐟调整呀

**Comment by ABA2396** [comment]
用的什么远程软件？我用向日葵和todesk都是有显示的

**Reply by TlgYuri** [reply]
我用这两个都没有显示😢我图省电把显示器关了，跟这个有关系吗

**Reply by ABA2396** [reply]
外置显示器关屏幕可能没关系，但要是是笔记本断开显示器直接让窗口不渲染了估计就不行了。还是说有开什么省电模式直接让无显示器的时候不渲染？

**Reply by TlgYuri** [reply]
我是台式外接的显示器，电源选项也是性能优先

**Reply by error404e2** [reply]
不能关闭显示屏

**Comment by ksqeib** [comment]
试试mstsc进行控制？

**Reply by SummerTau** [reply]
可以把todesk高级设置中D3D渲染关了试试

**Comment by phzhao12** [comment]
買個「顯示卡假負載」即可解決問題，假負載是讓你關閉顯示器時顯示卡依然能夠正常運作的好幫手，本人實測能夠有效解決MAA遠端操作白畫面的問題。

**Comment by acnekot** [comment]
Windows平台使用第三方远程操控（如Todesk），在显示屏关闭（没有识别）的情况下，显卡不会输出画面
可以换用微软的远程桌面，亲测可以正常输出画面

**Comment by Asatoshi314** [comment]
没有连接任何显示器的设备，可以装一个Zako VDD虚拟显示器保持常开，这种情况下不会白屏
